Disable so versioning since they are really not a versioned shared lib.

Upstream-Status: Submitted @ https://sourceforge.net/p/omxil/bugs/59/

Signed-off-by: Drew Moseley <drew_moseley@mentor.com>

diff -rub libomxil-bellagio-0.9.3-orig/src/components/audio_effects/Makefile.am libomxil-bellagio-0.9.3/src/components/audio_effects/Makefile.am
--- libomxil-bellagio-0.9.3-orig/src/components/audio_effects/Makefile.am	2014-07-20 15:22:00.858425234 -0400
+++ libomxil-bellagio-0.9.3/src/components/audio_effects/Makefile.am	2014-07-20 15:25:42.687525225 -0400
@@ -10,4 +10,5 @@
 libomxaudio_effects_la_CFLAGS = -I$(top_srcdir)/include \
 				-I$(top_srcdir)/src \
 				-I$(top_srcdir)/src/base
+libomxaudio_effects_la_LDFLAGS = -avoid-version
 
diff -rub libomxil-bellagio-0.9.3-orig/src/components/clocksrc/Makefile.am libomxil-bellagio-0.9.3/src/components/clocksrc/Makefile.am
--- libomxil-bellagio-0.9.3-orig/src/components/clocksrc/Makefile.am	2014-07-20 15:22:00.858425234 -0400
+++ libomxil-bellagio-0.9.3/src/components/clocksrc/Makefile.am	2014-07-20 15:24:49.151259753 -0400
@@ -10,4 +10,4 @@
                              -I$(top_srcdir)/include \
                              -I$(top_srcdir)/src \
                              -I$(top_srcdir)/src/base
-
+libomxclocksrc_la_LDFLAGS = -avoid-version
diff -rub libomxil-bellagio-0.9.3-orig/src/components/videoscheduler/Makefile.am libomxil-bellagio-0.9.3/src/components/videoscheduler/Makefile.am
--- libomxil-bellagio-0.9.3-orig/src/components/videoscheduler/Makefile.am	2014-07-20 15:22:00.862425254 -0400
+++ libomxil-bellagio-0.9.3/src/components/videoscheduler/Makefile.am	2014-07-20 15:22:36.462601786 -0400
@@ -6,7 +6,7 @@
 								library_entry_point.c
 
 libomxvideosched_la_LIBADD = $(top_builddir)/src/libomxil-bellagio.la
-libomxvideosched_la_LDFLAGS = 
+libomxvideosched_la_LDFLAGS = -avoid-version
 libomxvideosched_la_CFLAGS = -I$(top_srcdir)/include \
 			-I$(top_srcdir)/src \
 			-I$(top_srcdir)/src/base
diff -rub libomxil-bellagio-0.9.3-orig/src/dynamic_loader/Makefile.am libomxil-bellagio-0.9.3/src/dynamic_loader/Makefile.am
--- libomxil-bellagio-0.9.3-orig/src/dynamic_loader/Makefile.am	2014-07-20 15:22:00.862425254 -0400
+++ libomxil-bellagio-0.9.3/src/dynamic_loader/Makefile.am	2014-07-20 15:22:36.462601786 -0400
@@ -3,7 +3,7 @@
 omxdynamicloader_LTLIBRARIES = libomxdynamicloader.la
 libomxdynamicloader_la_SOURCES = ste_dynamic_component_loader.c ste_dynamic_component_loader.h
 
-libomxdynamicloader_la_LDFLAGS = -lomxil-bellagio -L$(top_builddir)/src/.libs
+libomxdynamicloader_la_LDFLAGS = -lomxil-bellagio -L$(top_builddir)/src/.libs -avoid-version
 libomxdynamicloader_la_CFLAGS = -I$(top_srcdir)/include \
 			-I$(top_srcdir)/src \
 			-I$(top_srcdir)/src/base \
diff -rub libomxil-bellagio-0.9.3-orig/src/Makefile.am libomxil-bellagio-0.9.3/src/Makefile.am
--- libomxil-bellagio-0.9.3-orig/src/Makefile.am	2014-07-20 15:22:00.862425254 -0400
+++ libomxil-bellagio-0.9.3/src/Makefile.am	2014-07-20 15:22:36.462601786 -0400
@@ -8,7 +8,7 @@
 omxregister_bellagio_CFLAGS = -DOMXILCOMPONENTSPATH=\"$(plugindir)/\" \
 			      -I$(top_srcdir)/include
 omxregister_bellagio_LDADD = $(lib_LTLIBRARIES)
-omxregister_bellagio_LDFLAGS = -lomxil-bellagio -L$(builddir)
+omxregister_bellagio_LDFLAGS = -lomxil-bellagio -L$(builddir) -avoid-version
 
 lib_LTLIBRARIES = libomxil-bellagio.la
 libomxil_bellagio_la_SOURCES = component_loader.h \
@@ -29,7 +29,7 @@
 libomxil_bellagio_la_CFLAGS = -I$(top_srcdir)/include -I$(srcdir)/base -I$(srcdir)/core_extensions \
                               -DINSTALL_PATH_STR=\"$(plugindir)\" -DOMX_LOADERS_DIRNAME=\"$(libdir)/omxloaders\/\"
 libomxil_bellagio_la_LIBADD = base/libomxbase.la core_extensions/libomxcoreext.la -lpthread
-libomxil_bellagio_la_LDFLAGS = -version-info @SHARED_VERSION_INFO@
+libomxil_bellagio_la_LDFLAGS = -avoid-version
 
 include_extradir = $(includedir)/bellagio
 
